library/core/class.request.php

```diff
@@ -990,8 +990,12 @@ public function url($path = '', $withDomain = false, $ssl = null) {
             $parts[] = '';
         }
 
-        if ($withDomain !== '/' && $this->webRoot() != '') {
-            $parts[] = $this->webRoot();
+        $webRoot = '';
```

Comment:
Calling webroot is costly. That's why I did it like it. Otherwise I could just keep the code like it was.
#microOptimizationBigNope